communications/entities/messages.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass, asdict, field
+from datetime import datetime
+from typing import List, Optional
+
+from communications.entities.base import BaseEntity
+
+
+@dataclass
+class Message(BaseEntity):
+    sender_id: int
+    receiver_id: int
+    content: str
+    read_at: Optional[datetime] = None
+
```

Comment:
add
    def mark_as_read(self):
        """Mark the message as read by setting the read_at timestamp."""
        self.read_at = datetime.now()